Formula/opam.rb

```diff
@@ -1,11 +1,13 @@
 class Opam < Formula
-  desc "Package manager for OCaml"
+  desc "The OCaml package manager"
   homepage "https://opam.ocaml.org"
-  url "https://github.com/ocaml/opam/archive/1.2.2.tar.gz"
-  sha256 "3e4a05df6ff8deecba019d885ebe902eb933acb6e2fc7784ffee1ee14871e36a"
-  revision 4
+  url "https://github.com/ocaml/opam/releases/download/2.0.0/opam-full-2.0.0.tar.gz"
+  sha256 "9dad4fcb4f53878c9daa6285d8456ccc671e21bfa71544d1f926fb8a63bfed25"
   head "https://github.com/ocaml/opam.git"
 
+  depends_on "glpk" => :build
```

Comment:
Sorry for jumping in, but I wonder if we need `glpk` to build since the release note for  2.0.0~beta5 says:
> Optimised use of the built-in solver, `glpk` is now also built in, hence not required to compile opam